Sources/SwiftGenKit/Parsers/Yaml/YamlParser.swift

```diff
@@ -22,37 +22,30 @@ public enum Yaml {
 
   // MARK: Yaml File Parser
 
-  public final class Parser: SwiftGenKit.Parser {
+  public class Parser: SwiftGenKit.Parser {
```

Comment:
Why no more `final`?

Reply:
See comment about the JSON parser (defaultFilter) above.